radicle/src/profile.rs

```diff
@@ -137,3 +142,18 @@ pub fn home() -> Result<PathBuf, io::Error> {
         ))
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct Paths<'a> {
+    home: &'a PathBuf,
```

Comment:
Would be better to store a `&'a Path` here

Reply:
fixed